components/forms/form.jsx

```diff
@@ -0,0 +1,207 @@
+                            </div>
+                        </div>
+                        <div className="flex flex-row border-2 border-black mx-6 mt-6 w-327px h-56px rounded-2xl box-border">
+                            <div>
+                                <svg className="my-3 mx-3" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <path d="M4 4H20C21.1 4 22 4.9 22 6V18C22 19.1 21.1 20 20 20H4C2.9 20 2 19.1 2 18V6C2 4.9 2.9 4 4 4Z" stroke="#18191F" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                                    <path d="M22 6L12 13L2 6" stroke="#18191F" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                                </svg>
+                            </div>
+                            <div>
+                                <input className="w-251px h-28px my-3" type="text" name="password" placeholder="Email address" />
+                            </div>
+                        </div>
+                        <div className="flex flex-row border-2 border-black mx-6 mt-6 w-327px h-56px rounded-2xl box-border">
+                            <div>
+                                <svg className="my-3 mx-3" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <path d="M19 11H5C3.89543 11 3 11.8954 3 13V20C3 21.1046 3.89543 22 5 22H19C20.1046 22 21 21.1046 21 20V13C21 11.8954 20.1046 11 19 11Z" stroke="#18191F" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                                    <path d="M7 11V7C7 5.67392 7.52678 4.40215 8.46447 3.46447C9.40215 2.52678 10.6739 2 12 2C13.3261 2 14.5979 2.52678 15.5355 3.46447C16.4732 4.40215 17 5.67392 17 7V11" stroke="#18191F" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                                </svg>
+                            </div>
+                            <div>
+                                <input className="w-251px h-28px my-3" type="password" name="password" placeholder="●●●●●●●" />
+                            </div>
+                        </div>
+                        <button className="mt-8 mx-6 w-327px h-60px rounded-2xl bg-yellow text-black border-2 border-black font-sans text-xl font-extrabold box-border shadow-lg ">Sign up</button>
+                        <div className="flex flex-row mx-6 mt-8 mb-6">
+                            <p className="text-black font-sans text-sm font-normal">Already have account?</p>
+                            <p className="text-red font-sans text-sm font-bold ml-1">Go here</p>
+                        </div>
+                    </div>
+                </form>
+            </div>
+
+            <div className="flex flex-col border-2 border-white bg-white w-375px h-224px m-5 items-stretch">
```

Comment:
This input should be inputtable 
![image](https://user-images.githubusercontent.com/17536627/124087726-78a7e100-da7c-11eb-8086-34e0c01907f0.png)